docs/index.rst

```diff
@@ -145,7 +145,7 @@ class SpotifyClientCredentials that can be used to authenticate requests like so
     playlists = sp.user_playlists('spotify')
     while playlists:
         for i, playlist in enumerate(playlists['items']):
-            print("%4d %s %s" % (i + 1 + playlists['offset'], playlist['uri'],  playlist['name']))
+            print("{:4d} {} {}".format(i + 1 + playlists['offset'], playlist['uri'],  playlist['name']))
```

Comment:
@dieser-niko I think we need to be consistent in our code. 
Most of files are using f-string instead of format.
You can replace this code to this:
```python
print(f"{i + 1 + playlists['offset']:4d} {playlist['uri']} {playlist['name']}")
```

Reply:
I only went through the .py files, looks like I missed that one